contracts/package.json

```diff
@@ -16,9 +16,9 @@
     "compile": "hardhat compile",
     "coverage": "hardhat coverage",
     "prepublishOnly": "pnpm compile && ./scripts/prepublish_generate_abi_folder",
-    "publish-beta": "pnpm publish --tag beta",
+    "publish- beta": "pnpm publish --tag beta",
     "publish-prod": "pnpm publish --tag latest",
-    "solhint": "solhint --max-warnings 0 \"./src/v0.8/**/*.sol\""
+    "solhint": "solhint --max-warnings 2 \"./src/v0.8/**/*.sol\""
```

Comment:
what are the warnings? if necessary, can they be suppressed inline?

Reply:
Not in this PR, I've asked RelEng to make the Solhint step required to merge so people cannot commit issues anymore

Reply:
huh they pushed a `KeystoneForwarder` that is very much WIP, smh